README.asciidoc

```diff
@@ -29,6 +29,8 @@ information. Depending on your sign-up date and how long you keep a certain
 level, you can get qutebrowser t-shirts, stickers and more!
 // QUTE_WEB_HIDE_END
 
+image:https://qmaury.com/images/banners/microbanners/quteget.gif["qutegetBanner",link="https://qmaury.com/images/banners/microbanners/quteget.gif"]
+
```

Comment:
Maybe it is too early and I am totally misreading this, but wouldn't this result in an image linking to itself? I'd expect to be taken to some sort of download page if I clicked on a banner like this.

Reply:
yes, maybe you have a point.
I didn't really think too much about where it really linked, I guess my main idea was just to have it there.
Here, lemme link that to the Qute download page :)